src/components/ProjectorWindow.js

```diff
@@ -8,18 +8,36 @@ const ProjectorWindow = () => (
         QUESTION
       </h1>
     </Alert.Heading>
-    <Jumbotron fluid>
-      <Container>
+    <Container fluid>
+      <Jumbotron fluid>
         <h1>
-          Fluid jumbotron
+          This is an example multiple choice question.
         </h1>
-        <p>
-          This is a modified jumbotron that occupies the entire horizontal space of
-          its parent.
-        </p>
-      </Container>
-    </Jumbotron>
-  </Alert>
+        <img src="https://fakeimg.pl/1280x720"
+          alt={"rich content"}
+          style={{ width: "auto", minHeight: "500px", maxHeight: "500px" }} />
+      </Jumbotron>
+
+      <Row>
+        <Col>
+          <Button block><h1>A</h1></Button>
+        </Col>
+        <Col>
+          <Button block><h1>A</h1></Button>
+        </Col>
+      </Row>
+      <Row>&nbsp;</Row>
+      {/* couldn't find a Row Gutter option above is to add vertical spacing */}
```

Comment:
Use `margin bottom` instead. Your code should be self-explanatory, avoid writing comments.

Reply:
how?
can't find a good method and what I have here is exactly how I need it to look.

Reply:
You have two ways: `style={{ marginBottom: '30px' }}` or `className='mb-4'`

Reply:
```suggestion
```